### DISCUSSION TITLE
[건의? & 협력?] 가상또는 시운전 열차의 grf 적용여부

### DISCUSSION THREAD

**Discussion by evepoi** [KoreanGRF/KoreanTrainSet#310] [일반(General)]
한국열차세트의 열차들이 실제 운영하는 열차를 위주로 제작하는 것으로 알고 있었습니다.
지금은 변경되셨는지는 모르겠습니다.

물론 본업이 있으신지라, 즉시 제작하는것도 쉬운 일이 아니실것으로 보여집니다.

본론으로 들어가서,
이슈 자료에 새로운 열차나 도색추가 열차의 한하여, YST에 선공개 하고자 합니다.
향후 한국열차세트가 새로 업데이트하면서 선공개된 열차가 있다면, YST에서는 제외하는 과정을 거치게 할 예정입니다.

하여, 한국열차세트 자료를 활용하는 협업?  으로 될듯하여 의견을 드려봅니다.

**Comment by telk5093** [comment]
1. 언제나 제1우선순위는 '현재 실 여객 영업을 하는 차량'입니다. 달라진 적은 없습니다.
 2. Issue에 올라온 그래픽 스프라이트는 아직 한국 열차 세트에 반영된 것이 아니므로 제 관할이 아닙니다. 각 그래픽 스프라이트 작업자에게 별도로 문의하세요.

**Reply by evepoi** [reply]
그래픽 작업자와 협의하여 제작할 수 있도록 해보겠습니다.  답변 감사합니다.
